diff --git a/deemon/__init__.py b/deemon/__init__.py
index ee56041..5ec2630 100644
--- a/deemon/__init__.py
+++ b/deemon/__init__.py
@@ -1,8 +1,4 @@
 #!/usr/bin/env python3
-from deemon.utils import startup
 
 __version__ = '2.22'
 __dbversion__ = '3.7'
-
-appdata = startup.get_appdata_dir()
-startup.init_appdata_dir(appdata)
diff --git a/deemon/__main__.py b/deemon/__main__.py
index 532f1a3..a041b0a 100644
--- a/deemon/__main__.py
+++ b/deemon/__main__.py
@@ -1,7 +1,10 @@
 from deemon import cli
+from deemon.utils import startup
 
 
 def main():
+    appdata = startup.get_appdata_dir()
+    startup.init_appdata_dir(appdata)
     cli.run()
 
 
