diff --git a/Telegram/SourceFiles/boxes/pin_messages_box.cpp b/Telegram/SourceFiles/boxes/pin_messages_box.cpp
index 3b9718ce..502da59f 100644
--- a/Telegram/SourceFiles/boxes/pin_messages_box.cpp
+++ b/Telegram/SourceFiles/boxes/pin_messages_box.cpp
@@ -65,14 +65,14 @@ void PinMessageBox::prepare() {
 				tr::now,
 				lt_user,
 				_peer->shortName()),
-			false,
+			true,
 			st::defaultBoxCheckbox);
 		_checkbox = _pinForPeer;
 	} else if (!_pinningOld && (_peer->isChat() || _peer->isMegagroup())) {
 		_notify.create(
 			this,
 			tr::lng_pinned_notify(tr::now),
-			true,
+			false,
 			st::defaultBoxCheckbox);
 		_checkbox = _notify;
 	}
