diff --git a/Telegram/SourceFiles/boxes/pin_messages_box.cpp b/Telegram/SourceFiles/boxes/pin_messages_box.cpp
index a2fdf4cbd..ac7f8c10a 100644
--- a/Telegram/SourceFiles/boxes/pin_messages_box.cpp
+++ b/Telegram/SourceFiles/boxes/pin_messages_box.cpp
@@ -59,7 +59,7 @@ void PinMessageBox(
 					tr::now,
 					lt_user,
 					peer->shortName()),
-				false,
+				true,
 				st::urlAuthCheckbox);
 			object->setAllowTextLines();
 			state->pinForPeer = Ui::MakeWeak(object.data());
@@ -68,7 +68,7 @@ void PinMessageBox(
 			auto object = object_ptr<Ui::Checkbox>(
 				box,
 				tr::lng_pinned_notify(tr::now),
-				true,
+				false,
 				st::urlAuthCheckbox);
 			object->setAllowTextLines();
 			state->notify = Ui::MakeWeak(object.data());
